app/views/devise/registrations/edit.html.erb

```diff
@@ -30,6 +30,12 @@
       <%= f.input :is_producer,
                   required: true,
                   autofocus: true %>
+      <%= f.input :current_password,
+                  hint: "we need your current password to confirm your changes",
+                  required: true,
+                  label: false,
+                  placeholder: 'Current password',
+                  input_html: { autocomplete: "current-password" } %>
```

Comment:
Comentário do André:

"People, I manage to solve your bug on the edit user.
The problem was that, someone removed the field that would be asking for the PASSWORD CONFIRMATION to actually update the user. The password field that you have in your form is not the one used to confirm your identity but the one used to UPDATE THE PASSWORD.....
Solution is to add this input back into the form."